src/main/java/frc/robot/Constants.java

```diff
@@ -198,8 +198,16 @@ public static class PhotonVisionConstants {
 
   public static class IntakeConstants {
     //TODO: Assign a real CAN ID
-    public static final int kIntakeMotorID = 0;
+    public static final int kTopIntakeMotorID = 0;
+    public static final int kBottomIntakeMotorID = 1;
     public static final int kIntakeCurrentLimit = 40;
+    public static final double kShaftDiameter = 0.0127;
+    public static final double kTopGearRatio = 0.83;
```

Comment:
This is hard to understand/audit. `15 / 18` would be clearer.

Reply:
This is fixed now